### ISSUE TITLE
Attach multiple policies on a user 

### ISSUE THREAD

**Issue by alexisdondon** [minio/minio-java#1484]
I don't see the possibility to attach multiple policy to a user, i ve seen setPolicy allowing one policy to give but it could be nice to add the possibility to give a List<String> policy to the user.

At least we could read the current list policy of the user and update it.

If not possible could the method ```` execute```` be public to allow adding some functionnality when using the minioAdminClient 

https://github.com/minio/minio-java/blob/7ca9500165ee13d39f293691943b93c19c31ebc2/adminapi/src/main/java/io/minio/admin/MinioAdminClient.java#L126

**Comment by balamurugana** [comment]
According to admin API, `setPolicy()` accepts only one policy. Not sure what API you are looking for. However you are free to send a PR and the PR will be merged based on approvals.

**Comment by alexisdondon** [comment]
i was looking for this https://min.io/docs/minio/linux/reference/minio-mc-admin/mc-admin-policy-attach.html#id1 mc admin attach policy can attach one or more policies to a user.

**Comment by harshavardhana** [comment]
You can pass comma separated values of your policy names.

**Comment by balamurugana** [comment]
> You can pass comma separated values of your policy names.

@harshavardhana currently https://github.com/minio/minio/blob/master/cmd/admin-handlers-users.go#L1613 API is implemented.  Are you saying this API supports multiple policies as comma separated values?

**Comment by harshavardhana** [comment]
Yes that is correct https://github.com/minio/minio/blob/master/cmd/iam-store.go#L193

**Comment by balamurugana** [comment]
@alexisdondon You would need to pass comma separated policy names as a single string to `policyName` argument.
